src/site/stages/build/drupal/graphql/nodeSupportResourcesDetailPage.graphql.js

```diff
@@ -0,0 +1,52 @@
+const entityElementsFromPages = require('./entityElementsForPages.graphql');
+
+const fragment = `
+fragment nodeSupportResourcesDetailPage on NodeSupportResourcesDetailPage {
+  ${entityElementsFromPages}
+  entityBundle
+  changed
+  title
+
+  fieldContentBlock {
+    entity {
+      entityType
+      entityBundle
+      ... wysiwyg
+      ... collapsiblePanel
+      ... process
+      ... qaSection
+      ... qa
+      ... listOfLinkTeasers
+      ... reactWidget
+      ... spanishSummary
+      ... alertParagraph
+      ... table
+      ... downloadableFile
+      ... embeddedImage
+      ... numberCallout
```

Comment:
I think we can remove some of this per the [template requirement](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/template-requirements.md#about) -

> No images, videos, subway maps, number callouts, etc.

Reply:
Does it hurt things to leave it in in case they want to add support for those entities in the future?